src/h5web/toolbar/controls/DomainSlider/Track.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ReactElement } from 'react';
+import type { AxisScale, Domain } from '../../../vis-packs/core/models';
+import styles from './DomainSlider.module.css';
+
+interface Props {
+  scale: AxisScale;
+  dataDomain: Domain;
+}
+
+function Track(props: Props): ReactElement {
+  const { scale, dataDomain } = props;
+
+  const [min, max] = dataDomain;
+  const cssScale = scale.copy().range([0, 100]) as AxisScale;
```

Comment:
Why do you need the cast here ?

Reply:
Because the types of the chained methods in D3 aren't great. `visx` doesn't go as far as fixing that, unfortunately.